README.md

```diff
@@ -1,17 +1,28 @@
 
-TotalFreedomMod is a CraftBukkit server plugin designed primarily to support the [Official TotalFreedom Minecraft Server](https://totalfreedom.me/). However, you are more than welcome to adapt the source for your own server.
+TotalFreedomMod (TFM) is a Spigot server plugin, designed primarily to support the [TotalFreedom Minecraft Server](https://totalfreedom.me/). However, you are more than welcome to adapt the source for your own server.
 
-This plugin was originally coded by StevenLawson (Madgeek1450), with Jerom van der Sar (Prozza) becoming heavily involved in its development some time later. It consists of over 85 custom coded commands and a large variety of distinguishable features not included in any other plugin. The plugin has since its beginning grown immensely. Together, with the main TotalFreedom server, TotalFreedomMod has a long-standing reputation of effectiveness whilst maintaining a clear feeling of openness towards the administrators and the players themselves.
+TotalFreedomMod consists of over 175 custom-coded commands, and a huge variety of unique features. For the better part of a decade, TFM has been trusted to provide TF with the administration tools, player utilities, community integration, and fun surprises that define the TotalFreedom experience. TFM is always growing to fulfill the community's wants and needs.
 
-### Contributing
+Originally coded by Steven Lawson (Madgeek1450) and Jerom van der Sar (Prozza), TotalFreedomMod is a community effort with a dedicated and passionate developer base.
 
-Please see [CONTRIBUTING.md](CONTRIBUTING.md) if you are interested in developing TotalFreedomMod.
+## Contributing
+Please see [CONTRIBUTING.md](CONTRIBUTING.md) if you are interested in developing TotalFreedomMod. Pull requests welcome!
 
-For information on how TotalFreedomMod is licensed, please see [LICENSE.md](LICENSE.md).
+Please see [LICENSE.md](LICENSE.md) for information on how TotalFreedomMod is licensed.
 
-For information on our security policy and reporting an issue, please see [SECURITY.md](SECURITY.md)
+Please see [SECURITY.md](SECURITY.md) for information on our security policy and how to report an issue.
 
-### Compiling
+## Compiling
+1) Clone this repository. If you would rather not use Git on the command line, GitHub Desktop [(Windows/macOS)](https://desktop.github.com/) [(Linux)](https://github.com/shiftkey/desktop) makes contributing easy.
+2) You will need an IDE for developing plugins. We recommend [IntelliJ IDEA](https://www.jetbrains.com/idea/). This plugin is built using Maven, which is included with IDEA.
```

Comment:
Only thing I'd say is I don't recommend IntelliJ to folks, I'm not sure if people on the team use it but I still recommenced folks to use NetBeans. I've personally had a lot of issues with IntelliJ before so it's not something I'd suggest we want to be pushing.